content/en/blog/2025/ambient-multicluster/index.md

```diff
@@ -0,0 +1,106 @@
+---
+title: Introducing multicluster support for ambient mode
+description: Introducing multicluster support for ambient mode.
+date: 2025-08-04
+attribution: Jackie Maertens (Microsoft), Keith Mattix (Microsoft), Mikhail Krinkin (Microsoft), Steven Jin (Microsoft)
+keywords: [ambient,multicluster]
+---
+
+Multicluster has been one of the most requested ambient features — and as of Istio 1.27, it's now available.
+We sought to capture the benefits and avoid the complications of multicluster architectures using the same modular design that ambient users love.
+While still in alpha, this release delivers the core functionality of a multicluster mesh and lays the groundwork for a full feature set in upcoming releases.
+
+## Multicluster's Many Benefits (and Challenges)
+
+Multicluster architectures increase outage resilience, shrink the blast radii,
+ease adoption of data residence policies, and simplify cost tracking.
+That said, integrating multiple clusters poses connectivity, security, and operation hurdles.
+
+In a single Kubernetes cluster, every pod can directly connect to another pod via a unique pod IP or service VIP.
+We lose these guarantees when we start thinking of multicluster architectures.
+IP address spaces of different clusters might overlap.
+Even if they didn't, nodes in one cluster would not know how to route traffic from one cluster to another.
+
+Establishing cross-cluster connectivity also presents security challenges.
+Cross-cluster connectivity means that pod-to-pod traffic can leave cluster boundaries -- and that pods may accept connections from outside the cluster.
+Without care, an attacker could connect to a vulnerable pod, or sniff unencrypted traffic.
+
+For a multicluster solution to be viable, it must at least securely connect clusters, and do so
+through APIs that are simple enough to keep pace with ever-changing environments.
+
+## Key Components.
+
+Ambient multicluster extends ambient with new components and minimal APIs to
+securely connect clusters using the same lightweight, modular architecture.
+
+### East-West Gateways
+
+Each cluster deploys an east-west gateway with a globally routable IP that acts as an entrypoint for cross-cluster communication.
+The east-west gateways are configured using GatewayAPI and controlled by istiod.
```

Comment:
```suggestion
The east-west gateways are configured using Gateway API and controlled by istiod.
```

Reply:
I also don't know how important it is to include this sentence. What is the user gaining from it?